library/save_utils.py

```
@@ -139,6 +142,317 @@ def load_train_state_metadata(input_dir: str) -> Optional[Dict[str, int]]:
+    train_util_module.save_and_remove_state_stepwise = save_and_remove_state_stepwise
+    train_util_module.save_state_on_train_end = save_state_on_train_end
+    train_util_module.resume_from_local_or_hf_if_specified = resume_from_local_or_hf_if_specified
+    huggingface_util_module.upload = _rank0_upload_only
+    train_util_module.huggingface_util.upload = _rank0_upload_only
+
+    if patch_train_network_upload:
+        try:
+            import train_network
+            train_network.huggingface_util.upload = _rank0_upload_only
+        except Exception:
+            pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)